# musl cp -rn behaves differently from gnu cp -rn
# the gnu version verifies on file level whether the target exists and musl on directory level only
diff -ruN a/Makefile b/Makefile
--- a/Makefile	2024-03-06 10:42:16.333816043 +0100
+++ b/Makefile	2024-03-06 10:42:34.613808357 +0100
@@ -95,11 +95,11 @@
 	find $(DESTDIR) -iname "*gpl*.txt" -exec rm -f {} \;
 
 js: $(JSDEPLOY)/fingerprint.js $(JSDEPLOY)/resize.js $(JSDEPLOY)/grommunio.js $(JSDEPLOY)/extjs-mod/extjs-mod.js $(JSDEPLOY)/extjs/ext-base-all.js $(DESTDIR)/client/third-party/ux-thirdparty.js $(DEPLOYPURIFYJS) $(JSDEPLOY)/filepreviewer/pdfjs/build/pdf.sandbox.js $(JSDEPLOY)/filepreviewer/pdfjs/build/pdf.worker.js $(JSDEPLOY)/filepreviewer/pdfjs/build/pdf.js $(JSDEPLOY)/filepreviewer/pdfjs/web/viewer.js $(JSDEPLOY)/filepreviewer/ViewerJS/ImageViewerPlugin.js $(JSDEPLOY)/filepreviewer/ViewerJS/MultimediaViewerPlugin.js $(JSDEPLOY)/filepreviewer/ViewerJS/ODFViewerPlugin.js $(JSDEPLOY)/filepreviewer/ViewerJS/UnknownFilePlugin.js $(JSDEPLOY)/filepreviewer/ViewerJS/viewer.js $(JSDEPLOY)/filepreviewer/ViewerJS/video-js/video.js
-	cp -rn client/tinymce $(DESTDIR)/client/
-	cp -rn client/tinymce-languages $(DESTDIR)/client/
-	cp -rn client/tinymce-plugins $(DESTDIR)/client/
-	cp -rn client/extjs $(DESTDIR)/client/
-	cp -rn client/filepreviewer $(DESTDIR)/client/
+	cp -r client/tinymce $(DESTDIR)/client/
+	cp -r client/tinymce-languages $(DESTDIR)/client/
+	cp -r client/tinymce-plugins $(DESTDIR)/client/
+	cp -r client/extjs $(DESTDIR)/client/
+	cp -r client/filepreviewer $(DESTDIR)/client/
 	rm $(DESTDIR)/client/extjs/ext-base.js $(DESTDIR)/client/extjs/ext-base-debug.js $(DESTDIR)/client/extjs/ext-all.js $(DESTDIR)/client/filepreviewer/pdfjs/web/debugger.js
 
 $(DESTDIR)/%.php: %.php
